models/reporter.go

```diff
@@ -52,15 +52,17 @@ SELECT
 {{ .Group }} AS dimension,
 {{ if eq .Group "queryid" }} any(fingerprint) {{ else }} '' {{ end }} AS fingerprint,
 {{range $j, $col := .CommonColumns}}
-	SUM(m_{{ $col }}_cnt) AS m_{{ $col }}_cnt,
-	SUM(m_{{ $col }}_sum) AS m_{{ $col }}_sum,
-	MIN(m_{{ $col }}_min) AS m_{{ $col }}_min,
-	MAX(m_{{ $col }}_max) AS m_{{ $col }}_max,
-	AVG(m_{{ $col }}_p99) AS m_{{ $col }}_p99,
+    SUM(m_{{ $col }}_cnt) AS m_{{ $col }}_cnt,
+    SUM(m_{{ $col }}_sum) AS m_{{ $col }}_sum,
+    m_{{ $col }}_sum/m_{{ $col }}_cnt AS m_{{ $col }}_avg,
```

Comment:
should we do it for all metrics? looks like we can check if it's time metric or not

Reply:
Good point. I'll check.

Reply:
Changed